WordPress/twentytwentyfive
# Fix: Remove top and bottom margin 0 on full width patterns

**Description**


Fixes https://github.com/WordPress/twentytwentyfive/issues/328

This PR removes the margin top/bottom set to zero for full-width patterns.
It also updates the landing pages to be contained by a group with a block gap of zero.

Bonus:

- Add description to testimonials patterns.
- Indent several patterns.

**Screenshots**

N/A

**Testing Instructions**

1. Create a page.
2. Add the following patterns
   - Banner with book description
   - Banner with description and images grid
   - Clients section
   - Centered link and social links
   - Contact: Info and locations
   - Cover with big heading
   - Call to action with book links
   - Call to action with locations
   - Pricing two columns
   - Events – List with Call to Action
   - Events three columns with event images and titles
   - Grid with videos
   - Heading and paragraph with image on the right
   - Hero Centered Heading
   - Hero podcast
   - Link in bio with tight margins
   - Newsletter Sign Up
   - Poster-like section
   - Services - Subscriber only section
   - Services - Three-column boxed pricing table
   - Services - Three Columns
   - Review with large image on right
   - Three column layout with 6 testimonials
   - Two columns with avatar
   - FAQs
   - Two Headings and a List
3. Confirm that these patterns no longer have a margin top/bottom set to zero.
